tensorrt_llm/llmapi/llm_args.py

```diff
@@ -1596,6 +1596,26 @@ class TorchCompileConfig(BaseModel):
         "When torch compile is enabled, userbuffers is enabled by default.")
 
 
+class MoeConfig(BaseModel):
+    """
+    Configuration for MoE.
+
+    """
+    moe_backend: str = Field(default='CUTLASS',
```

Comment:
Since MoeConfig already contains the "Moe" keyword, maybe we can remove the "moe_" prefix for each field. The usage code will be changed from `moe_config.moe_backend` to `moe_config.backend`. So are other fields.